contracts/root/StakeManager.sol

```diff
@@ -202,23 +201,24 @@ contract StakeManager is StakeManagerInterface, RootChainable, Lockable {
       value >> 160;
       address newStaker = address(uint160(value));
       stakers[newStaker].activationEpoch = stakers[msg.sender].deActivationEpoch;
+      totalValidatorStake[currentEpoch + dynasty*2] = totalValidatorStake[currentEpoch + dynasty*2] + stakers[newStaker].amount;
```

Comment:
You need to update with `totalValidatorStake[currentEpoch]` too. Otherwise it will give invalid amount.